spec.md

```diff
@@ -541,6 +543,10 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+The registry SHOULD support at least `n=100` tags in a single response.
```

Comment:
can you link to the reason 100 is the should here..

Reply:
It's arbitrary. I'm happy to adjust it to another value.

Reply:
crane uses [1000 by default](https://github.com/google/go-containerregistry/blob/dbcd01c402b2f05bcf6fb988014c5f37e9b13559/pkg/v1/remote/options.go#L108) which apparently works with all current registries so maybe we can make this n=1000.

Reply:
It looks like quay.io actually caps out at 50, I thought they were more like 250, so I may have picked a number that's too large for current registries. 

```
curl -sSL -H 'Accept: application/json' https://quay.io/v2/openshift-release-dev/ocp-release/tags/list?n=1000  | jq .tags
```

Reply:
Is it really worth specifying a minimum?

Reply:
The "SHOULD" means it's advice that well behaved registries would follow, but there's no enforcement. The goal is to get some guidance that most implementations agree with so that servers and clients aren't both picking arbitrary numbers. Increasing this to 1000 if no additional major registries are excluded feels reasonable.

Reply:
I removed the recommended limit from the PR.